CMakeLists.txt

```diff
@@ -331,7 +330,6 @@ IF(NOT CYCLUS_DOC_ONLY)
         include(UseCython)
         find_package (Python3 REQUIRED COMPONENTS NumPy)
         find_package(Jinja2 REQUIRED)
```

Comment:
Is jinja also run-time only?

Reply:
No, `jinja2` is used by `gentypesystem.py` during the build process